packages/components/src/EditableText/EditableText.scss

```diff
@@ -46,6 +46,8 @@ $tc-circle-editable-text-size: 16px !default;
 		padding-left: $padding-small;
 		padding-right: $padding-small;
 		flex: 0 0 auto;
+		line-height: inherit;
+		min-height: inherit;
```

Comment:
Could you try with IE 11, please?
https://caniuse.com/#feat=css-unset-value
Just to be sure